code/modules/clothing/outfits/ert.dm

```diff
@@ -563,3 +563,49 @@
 	head = /obj/item/clothing/head/beret/militia
 	l_hand = /obj/item/megaphone
 	suit_store = /obj/item/gun/energy/laser/musket/prime
+
+/datum/outfit/centcom/ert/medical_commander
+	name = "Chief EMT"
+	id = /obj/item/card/id/advanced/centcom/ert/medical
+	suit = /obj/item/clothing/suit/toggle/labcoat/cmo
```

Comment:
I can't help but feel like these lads should have the medical ERT MODsuits, despite being non-combatants:
1. Very identifiable as ERT members at a glance
2. Avoids people from seeing them and think they're just some random doctor at a glance
3. MODsuits have a few modules which can elicit faster revival (in built defibrillator, thread ripper, surgical processor) (though I don't think the existing medical MOD has these, you might need a subtype)  
4. Is it even an ERT if they don't have the dorky looking suits

Reply:
1: I considered rectifying this with an alternate uniform, but I'm no spriter and if I waited long enough to commission something, the spark would be lost and I'd move on to something else.

2: No arguments here, that makes perfect sense to me.

3: Serious response to what is probably a joke point, but I figured the scrubs/armor for the commander would help emphasize how much the focus is on healing/revival over being a combatant. To help distinguish them from the standard ERT, making them flimsy and weak enough to understand that they're meant to remain strictly in the support realm.

I'm convinced, the MODsuit idea would probably work out better conceptually. Gimme some time to cook.

Reply:
Now, visually, we do have a semi-unused MOD suit skin for the combat medic modsuit, which doesn't get much love or play. Goes against your 3rd point but food for thought

Reply:
Space suits are usually essentially for ERTs - rarely have I sent one into a station with no holes. That's usually one of the biggest barriers to getting people revived in the round as well. I can imagine myself spawning space suits for this ERT the majority of times I would send them. Otherwise this is based, I've often done it manually by spawning a regular ERT and then select equipment all of them to medical.